resources/metadata/helmExecute.yaml

```diff
@@ -0,0 +1,371 @@
+metadata:
+  name: helmExecute
+  aliases:
+    - name: deployToKubernetes
```

Comment:
not sure if we should keep this alias.
Do you see a reason why we should have this alias?

Reply:
changed it